src/components/Timeline/Event.js

```diff
@@ -60,19 +60,20 @@ Event.propTypes = {
 
 export const mapStateToEventProps = (state, ownProps) => {
     const event = ownProps.event
-    const eventType = state.eventTypes.records[ownProps.event.eventTypeId]
+    const eventType = state.eventTypes.records[event.eventTypeId]
     const ticket = state.tickets.map[ownProps.ticketId]
     const engagement = state.engagements.list.find(engagement => engagement.id === ownProps.engagementId)
     return {
         ...ownProps,
         ticket,
         engagement,
         eventId: event.id,
-        time: moment(event.occurred),
+        eventTypeId: event.eventTypeId,
+        time: moment(event.occurred ? event.occurred: event.Occurred),
```

Comment:
What is the difference between `event.occurred` and `event.Occurred`?

Reply:
JSON serialization inconsistencies between SignalR and the REST API on the Gateway. Once that's fixed in the Gateway, we should revert this line.